src/components/DuosHeaderOld.js

```diff
@@ -0,0 +1,746 @@
+import {Component, Fragment, useState} from 'react';
```

Comment:
This entire file is the original untouched `DuosHeader.js` file, which we can get rid of later